lib/cog/util/command_helpers.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Cog.Command.Helpers do
```

Comment:
There is actually already a module called `Cog.Command.Helpers`. I created when I did the which command. On another note, I generally like to have module names match their location in the file system. For example, `Cog.Command.Helpers` would live at `lib/cog/command/helpers.ex`.